httpd-ldapauth-proxy/templates/config.yaml

```diff
@@ -3,7 +3,8 @@ kind: ConfigMap
 metadata:
   name: httpd
 data:
-  httpd.conf: |
+  httpd.conf: |-
```

Comment:
Why are you manually adding new lines before and after, and telling Yaml to remove all new lines after the text?

Are you sure that the last statement of the configuration will work correctly if there are no new lines at the end of file ? (I know for sure that gpg doesn't accept an armored input when there is no new line at the end)